l10n_cl_financial_indicators/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Chilean Financial Indicators",
+    "version": "12.0.1.0.0",
+    "category": "Tools",
+    "development_status": "Production/Stable",
+    "maintainers": ["danisan"],
+    "license": "LGPL-3",
+    "author": "Blanco Martin & Asociados, Odoo Community Association (OCA)",
+    "website": "http://blancomartin.cl",
+    "depends": ["base"],
+    "data": [
+        "data/decimal.precision.xml",
```

Comment:
You should add decimal_precision dependency.

Reply:
Hello Daniel.

Solved at f150b8

Do you need a rebase?